clusterdock/topologies/cdh/actions.py

```diff
@@ -162,18 +198,48 @@ def start(args):
                 deployment.cluster.delete_service(service.name)
 
     hue_server_host_port = get_host_port_binding(primary_node.container_id, HUE_SERVER_PORT)
+    is_hue_installed = False
     for service in deployment.cluster.get_all_services():
         if service.type == 'HUE':
             logger.info("Once its service starts, Hue server will be accessible at http://%s:%s",
                         getfqdn(), hue_server_host_port)
+            is_hue_installed = True
             break
 
-    logger.info("Deploying client configuration...")
-    deployment.cluster.deploy_client_config().wait()
+    if args.kerberos:
+        logger.info('Updating config for kerberos...')
+        kerberos_config = {'SECURITY_REALM': args.network.upper(),
```

Comment:
Since you're using `args.network.upper()` a few times, may make a local variable called `realm` to hold it and keep things tidy?